### PR TITLE
Disable high fanout routing when nearly done routing [BUILD TEST]

### PR DESCRIPTION
**Pull request by HackerFoo**

### CODE REVIEW

File: vpr/src/route/route_timing.cpp [hunk 1]
```diff
@@ -735,6 +739,12 @@ bool try_timing_driven_route(const t_router_opts& router_opts,
         if (router_opts.congestion_analysis) profiling::congestion_analysis();
         if (router_opts.fanout_analysis) profiling::time_on_fanout_analysis();
         // profiling::time_on_criticality_analysis();
+
+        // clear router_iteration_stats and set .no_progress for the next iteration
+        size_t connections_routed_this_iteration = router_iteration_stats.connections_routed;
+        router_iteration_stats = {};
+        router_iteration_stats.no_progress = !(connections_routed_this_iteration < connections_routed_last_iteration);
```
Comment (litghost) [inline vpr/src/route/route_timing.cpp]: Why not just have a `no_progress` boolean, default to false, and once true, leave it true?  This also avoids the shuffling of the router_iteration_stats structure?

Comment (HackerFoo) [reply vpr/src/route/route_timing.cpp]: Are you suggesting a global? I don't think that's a good idea. This value is used in `timing_driven_route_sink()`.

**Comment by litghost** [comment]
@HackerFoo Can this PR be closed?

**Comment by HackerFoo** [comment]
Yeah.

**Pull request closed without merging**